bower.json

```diff
@@ -17,7 +17,7 @@
     "animate.css": "~3.4.0",
     "clipboard": "~1.5.5",
     "jquery.scrollTo": "~2.1.2",
-    "patternfly": "3.6.0",
+    "patternfly": "f012b78c7f90fada15718705a6d99f72d7060d5c",
     "wowjs": "~1.1.2"
```

Comment:
Right. These changes will cause the build to fail without the latest patternfly, so we either need to create a tag or reference a commit. We don't want to wait until the end of the sprint to cut a new patternfly release. Personally, I prefer a commit because we don't need to create anything new.